enginetest/queries.go

```diff
@@ -1724,14 +1724,6 @@ var QueryTests = []QueryTest{
 		`SELECT NOW() - NOW()`,
 		[]sql.Row{{int64(0)}},
 	},
-	{
-		`SELECT DATETIME() - NOW()`,
-		[]sql.Row{{int64(0)}},
-	},
-	{
-		`SELECT TIMESTAMP() - NOW()`,
```

Comment:
0 argument versions not part of mysql

https://dev.mysql.com/doc/refman/8.0/en/date-and-time-functions.html#function_timestamp

Reply:
done